lib/torrent.js

```diff
@@ -509,6 +505,22 @@ Torrent.prototype._onMetadata = function (metadata) {
     return new File(self, file)
   })
 
+  // Only select specified files (BEP53)
+  if (self.so) {
+    var specificFiles = parseRange.parse(self.so)
```

Comment:
Can we name this variable `selectOnlyFiles`, since "so" stands for "select only"?

Reply:
Of course, that would also make it more clear for people searching the code.